GLOSSARY.md

```diff
@@ -0,0 +1,44 @@
+# Parts of Class Names
+
+| Name        | Meaning                                                                                                                                                                                                        |
+| ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Abstract... | An incomplete class used for *dynamic* polymorphism. It has pure virtual methods that need to be implemented by subclasses.                                                                                    |
```

Comment:
Suggestion: `Abstract*`, `Base*`, `*Operator`, etc. ? It looks nicer :)

Reply:
done